.circleci/config.yml

```diff
@@ -22,6 +23,10 @@ jobs:
       - run: |
           $NYC yarn test
           curl -s https://codecov.io/bash | bash
+  node-10:
+    <<: *test
+    docker:
+      - image: oclif/nsis:10.10.0-18
```

Comment:
This image won't work. The node 11 `nsis` one did. We likely need to rebuild a new image with the node 10 but based on our current version. If we want to test node 10 I think I know where to tweak the Dockerfile.